packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -311,6 +311,15 @@ This test is testing this payload is returned from the server
 "hello world"
 ```
 
+### Azure_Lro_PollingSuccess
+
+- Endpoints:
+  - `put /lro/basic/put`
+  - `put /lro/basic/put/polling`
+  - `put /lro/basic/put`
+
+undefined
```

Comment:
@timotheeguerin  it seems that `pnpm generate-scenarios-summary` does not work well when we group several endpoints into one scenario

Reply:
hhm, I'll look into it, lets not block merging on this unless I fix it before